application/src/main/kotlin/net/corda/v5/application/messaging/FlowSession.kt

```diff
@@ -119,8 +112,9 @@ inline fun <reified R : Any> FlowSession.sendAndReceive(payload: Any): Untrustwo
 /**
  * Suspends until the counterparty sends us a message of type [R].
  *
- * Remember that when receiving data from other parties the data should not be trusted until it's been thoroughly verified for consistency
- * and that all expectations are satisfied, as a malicious peer may send you subtly corrupted data in order to exploit your code.
+ * Remember that when receiving data from other parties the data should not be trusted until it's been thoroughly
+ * verified for consistency and that all expectations are satisfied, as a malicious peer may send you subtly corrupted
+ * data in order to exploit your code.
  */
```

Comment:
`@param` etc (just copy what you use for the others, minus referencing `receiveType`).

Reply:
^